Dockerfile

```diff
@@ -1,5 +1,9 @@
+ARG TENTACLES_URL_TAG
+
 FROM python:3.8-slim-buster AS base
 
+ARG TENTACLES_URL_TAG
```

Comment:
`ARG TENTACLES_URL_TAG`

twice in dockerfile ?

Reply:
It's mandatory for multistage builds